webpack/scenes/Subscriptions/Details/SubscriptionDetails.js

```diff
@@ -58,75 +58,74 @@ class SubscriptionDetails extends Component {
     }
 
     return (
-      <div id="subscription-details">
-        {!subscriptionDetails.loading &&
-          <BreadcrumbsBar
+      <div style={{ margin: '24px' }}>
+        <div id="subscription-details">
```

Comment:
I see we already have a `SubscriptionDetails.scss` file alongside this one. And the `#subscription-details {` in there indicates that the file applies to this already-existing `<div>` with `id="subscription-details"`.

In general it's preferable to add CSS in `scss` files when possible, instead of style tags. I think maybe we could get rid of the outer div (and style tag) and instead just add a new `margin: 24px;` entry in `SubscriptionDetails.scss`. Thoughts?

Reply:
That sounds ok to me :+1:

Reply:
Done!